src/componentRenderer.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { renderToString } from 'react-dom/server'
+import { uniqueId } from 'lodash'
+
+import {
+  ServerStyleSheet,
+  __DO_NOT_USE_OR_YOU_WILL_BE_HAUNTED_BY_SPOOKY_GHOSTS
+} from 'styled-components'
+
+// For Jest. See: https://github.com/styled-components/styled-components/issues/1692
+const { StyleSheet } = __DO_NOT_USE_OR_YOU_WILL_BE_HAUNTED_BY_SPOOKY_GHOSTS
+StyleSheet.reset(true)
+
+const modes = {
+  CLIENT: 'client',
+  SERVER: 'server'
+}
+
+export function componentRenderer(config) {
+  const {
+    mode = modes.SERVER,
+    modules,
+    serialize = x => x
+    //
```

Comment:
#minor Left over comment delimiter?

Reply:
This is to keep Prettier from collapsing the API config onto one line; I could likely fix this by renaming the `x` placeholder value tho